go/error-handling/error_handling.go

```diff
@@ -0,0 +1,34 @@
+//This package provides functions
+package erratum
+
+// Use takes a ResourceOpener function as a parameter and attempts to
+// call it to obtain a resource. It resists TransientError generated by
+// this call.
+// It also uses this resource's Frob function, and repairs FrobErrors
+// generated by it by calling its Defrob function.
+func Use(o ResourceOpener, inp string) (returnedError error) {
+	returnedError = nil
```

Comment:
I don't think that this is needed as the default value for error is `nil`.

Reply:
Arf, yeah, eventually, I'll learn to start trusting the language on initialization values ^^. Thanks !